### PR TITLE
[DigitalOcean] Show load balancer id in logs

### PR DESCRIPTION
**Pull request by srikiz**
The [e2e tests](https://testgrid.k8s.io/sig-cluster-lifecycle-kops#e2e-kops-do-calico) for DO are failing to create a load balancer. I haven't been able to reproduce it locally, so adding the load balancer ID in logs for further troubleshooting.

FYI - @timoreimann

### CODE REVIEW

File: upup/pkg/fi/cloudup/dotasks/loadbalancer.go [hunk 1]
```diff
@@ -163,6 +163,8 @@ func (_ *LoadBalancer) RenderDO(t *do.DOAPITarget, a, e, changes *LoadBalancer)
 		return err
 	}
 
+	klog.Infof("Load balancer created with ID=%s", loadbalancer.ID)
```
Comment (olemarkus) [inline upup/pkg/fi/cloudup/dotasks/loadbalancer.go:166]:
```suggestion
	klog.V(2).Infof("Load balancer created with ID=%s", loadbalancer.ID)
```

**Review by olemarkus — CHANGES_REQUESTED** [review]
Maybe consider adding this to a higher verbosity.

File: upup/pkg/fi/cloudup/dotasks/loadbalancer.go (hunk 1 above)
Comment (srikiz) [reply upup/pkg/fi/cloudup/dotasks/loadbalancer.go:166]: @olemarkus - Strangely, I don't see the logs emitted if I use `klog.V(2)` when testing locally.
However, I do see if with `klog.Infof` as seen below

`I1114 10:36:38.306548    5486 loadbalancer.go:166] Load balancer created with ID=59a14963-08b5-4cfb-94cd-2ffd00d9b6e1`

Am I missing something here?

**Comment by srikiz** [comment]
Figured out the issue and created a separate PR - https://github.com/kubernetes/kops/pull/12758 to fix it. 
Hence, closing this.

**Pull request closed without merging**